### PR TITLE
removed unneccesary border; Fixes #529

### LINKED ISSUE #529: Remove left and right borders on Notes sidebar panels
The left and right borders are not needed screen shot 2017-12-13 at 2 52 27 pm

### PR DESCRIPTION
**Pull request by shikhar-scs**
As mentioned by @ryanfeeley and @vladikoff at #529, the unnecessary border has been fixed.
Please review

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ web-ext-artifacts/
 test/dist/*
 ckeditor-build/build/*
 build/
+,idea
```
Comment (vladikoff) [inline .gitignore]: Please do not add this, add this rule to your global git ignore

Comment (shikhar-scs) [reply .gitignore]: Yeah Sure @vladikoff . Apart from this is it fine ?

**Comment by Natim** [comment]
screen-capture

**Comment by Natim** [comment]
other-note

**Pull request merged**